scripts/src/java/org/oppia/android/scripts/todo/TodoCheck.kt

```diff
@@ -0,0 +1,302 @@
+package org.oppia.android.scripts.todo
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import org.oppia.android.scripts.proto.TodoExemption
+import org.oppia.android.scripts.proto.TodoExemptions
+import org.oppia.android.scripts.todo.data.Issue
+import org.oppia.android.scripts.todo.data.Todo
+import java.io.File
+import java.io.FileInputStream
+
+/**
+ * Script for ensuring that all TODOs present in the repository are correctly formatted and
+ * corresponds to an open issue on GitHub.
```

Comment:
```suggestion
 * correspond to open issues on GitHub.
```

Ditto elsewhere in the PR where this grammar needs to be updated.

Reply:
Done.